.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }} ${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
```

Comment:
**Performance Issue**: The environment variable "OPTIONS" includes the flag "--static_analysis.fb_infer.enabled=True" which enables static analysis with Facebook Infer. While static analysis is crucial for identifying potential issues, it can significantly increase the build time, especially for large projects. This can slow down the CI pipeline, impacting the overall developer productivity and efficiency. <br> **Fix**: Consider running static analysis as a separate, non-blocking step in your CI pipeline or on a scheduled basis rather than on every pull request. This can help in balancing the need for code quality with the need for fast build times. Alternatively, you can opt for incremental analysis or limit the scope of analysis to only the changed files to reduce the performance impact. <br> **Code Suggestion**: 
 ```
 jobs:
   code_review_job:
     steps:
       - name: Separate Static Analysis Step
         run: |
           if [ ${{ github.event_name == 'pull_request' }} ]; then
             echo "Running static analysis..."
             # Add static analysis tool commands here
           fi
 ```